src/main/java/org/reaktivity/nukleus/oauth/internal/stream/OAuthProxyFactory.java

```diff
@@ -397,6 +431,8 @@ private OAuthAccessGrant lookupGrant(
         private String subject;
         private long authorization;
         private long expiresAt;
+        private long challengeResponseTimeout;
```

Comment:
Rename to `challengeResponseTimeoutSeconds` to make the units clear, or `challengeResponseTimeoutMillis` if you've already converted it, which is better.